flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -277,8 +277,14 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
         }
     }
 
-    private func loadImageList(result: @escaping FlutterResult) {
-      let debugImages = PrivateSentrySDKOnly.getDebugImages() as [DebugMeta]
+    private func loadImageList(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+      guard let imageAddresses = call.arguments as? Set<String> else {
+          print("Arguments is not a list of string")
+          result(FlutterError(code: "X", message: "Cannot get image for address", details: nil))
```

Comment:
@buenaflor is there a list for the error codes? Which code should I use?

Reply:
I don't think we should send back an error, if it fails for whatever reason let's just fall back to the previous way of getting the debug image addresses